src/language-js/index.js

```diff
@@ -132,6 +133,7 @@ const printers = {
 };
 
 module.exports = {
+  options,
```

Comment:
`options` should probably be a property of a specific printer rather than a plugin, as a plugin can provide multiple languages/printers. In this case it should be `printers.estree.options`